components/script/dom/oscillatornode.rs

```diff
@@ -113,6 +114,24 @@ impl OscillatorNodeMethods for OscillatorNode {
     fn Detune(&self) -> DomRoot<AudioParam> {
         DomRoot::from_ref(&self.detune)
     }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn Type(&self) -> OscillatorType {
+        self.oscillator_type.get()
+    }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn SetType(&self, type_: OscillatorType) -> ErrorResult {
+        self.oscillator_type.set(type_);
+        if type_ == "custom" {
+            return Err(Error::InvalidState);
+        }
+        self.source_node
+            .node()
+            .message(AudioNodeMessage::OscillatorNode(
```

Comment:
```
error[E0599]: no variant or associated item named `OscillatorNode` found for type `servo_media_audio::node::AudioNodeMessage` in the current scope
   --> components/script/dom/oscillatornode.rs:131:40
    |
131 |             .message(AudioNodeMessage::OscillatorNode(
    |                                        ^^^^^^^^^^^^^^ variant or associated item not found in `servo_media_audio::node::AudioNodeMessage`
```

Updating `servo-media` will get the `AudioNodeMessage::OscillatorNode` variant as well.